From 0eb4dc650548be6de91e907e8bbffdb51e826e34 Mon Sep 17 00:00:00 2001
From: Muthukumar K <muthukumar.kandasamy@intel.com>
Date: Wed, 25 Jul 2012 16:09:44 +0545
Subject: [PORT FROM MAIN][JellyBean] Enable WiDi support in Audio middleware

BZ: 48525

Widi Support is enabled in the audio Middleware path
and audio HAL. This doesnot include Widi stack.

Change-Id: I19bc87973e6dcf02f563d98c59be3b21853db81f
Orig-Change-Id: I13744d1eff3d1f83eff9381b91b0961bb4b8640c
Signed-off-by: Muthukumar K <muthukumar.kandasamy@intel.com>
---
 include/system/audio.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/include/system/audio.h b/include/system/audio.h
index f1a7d66..f5a3703 100644
--- a/include/system/audio.h
+++ b/include/system/audio.h
@@ -321,6 +321,7 @@ enum {
                                  AUDIO_DEVICE_OUT_USB_ACCESSORY |
                                  AUDIO_DEVICE_OUT_USB_DEVICE |
                                  AUDIO_DEVICE_OUT_REMOTE_SUBMIX |
+                                 AUDIO_DEVICE_OUT_WIDI |
                                  AUDIO_DEVICE_OUT_DEFAULT),
     AUDIO_DEVICE_OUT_ALL_A2DP = (AUDIO_DEVICE_OUT_BLUETOOTH_A2DP |
                                  AUDIO_DEVICE_OUT_BLUETOOTH_A2DP_HEADPHONES |
-- 
1.7.4.1

